utilities/utility.py

```diff
@@ -81,6 +81,16 @@ def validate_date_time_range(**kwargs):
         )
 
 
+def validate_missing_items_field(**kwargs):
+    """
+    Function to validate the missing item field
+    when saving a check question response
+    :params kwargs
+    """
+    if 'missing_items' not in kwargs:
```

Comment:
Would it be possible to be specific in your validation instead of generalizing items?